rustls/src/crypto/cipher.rs

```diff
@@ -47,8 +105,9 @@ fn unpad_tls13(v: &mut Vec<u8>) -> ContentType {
 }
 
 /// Objects with this trait can encrypt TLS messages.
-pub(crate) trait MessageEncrypter: Send + Sync {
-    fn encrypt(&self, m: BorrowedPlainMessage, seq: u64) -> Result<OpaqueMessage, Error>;
+pub trait MessageEncrypter: Send + Sync {
+    /// Encrypt the message `msg`.
```

Comment:
Should this docstring mention the `seq` arg or do you think it's contextual enough to understand as-is?